README.rst

```diff
@@ -15,6 +15,11 @@ A great cloud needs great tools; we're excited to introduce *Azure CLI 2.0 - Pre
 Installation
 ===============
 
+Developer Setup
+^^^^^^^^^
+If you would like to setup a development environment and contribute to the CLI, see the following document:
```

Comment:
nit: In the sentence "If you would like to setup a ...", I believe it should be "set up" because it's being used as a verb?